packages/@sanity/form-builder/src/inputs/PortableText/toolbar/Toolbar.tsx

```diff
@@ -80,6 +81,13 @@ const InnerToolbar = memo(function InnerToolbar({
 
   const collapsed = rootElementRect ? rootElementRect?.width < 400 : false
 
+  useRovingFocus(
+    {
+      rootElement: rootElement,
+    },
+    [showActionMenu, showInsertMenu, collapsed]
```

Comment:
what happens when you pass dependencies here? what does it run when they change?

Reply:
Good point. I thought it would be needed, but I think I was wrong. I can remove these dependencies!